Failed make install

The install fails when I try to make install in the way described in the README.

`BUILD_PATH` seems to have been changed by adding `--arch arm64 --arch x86_64`. (https://github.com/yonaskolb/Mint/pull/191)

`.build/apple/Products/Release/mint`

```
$ git clone https://github.com/yonaskolb/Mint.git
$ cd Mint
$ make
swift build --disable-sandbox -c release --arch arm64 --arch x86_64
Fetching https://github.com/mxcl/Version.git
Fetching https://github.com/kylef/Spectre.git
Fetching https://github.com/kylef/PathKit.git
Fetching https://github.com/onevcat/Rainbow.git
Fetching https://github.com/jakeheis/SwiftCLI.git
Cloning https://github.com/kylef/PathKit.git
Resolving https://github.com/kylef/PathKit.git at 1.0.0
Cloning https://github.com/kylef/Spectre.git
Resolving https://github.com/kylef/Spectre.git at 0.9.0
Cloning https://github.com/jakeheis/SwiftCLI.git
Resolving https://github.com/jakeheis/SwiftCLI.git at 6.0.1
Cloning https://github.com/mxcl/Version.git
Resolving https://github.com/mxcl/Version.git at 1.1.2
Cloning https://github.com/onevcat/Rainbow.git
Resolving https://github.com/onevcat/Rainbow.git at 3.1.5
0%: CreateBuildDirectory /Users/nanashiki/Develop/tmp/Mint/.build/apple/Intermediates.noindex
0%: CreateBuildDirectory /Users/nanashiki/Develop/tmp/Mint/.build/apple/Products
0%: Write Version.modulemap
0%: Write SwiftCLI.modulemap
0%: Write Rainbow.modulemap
0%: Write Version-OutputFileMap.json (x86_64)
0%: Write PathKit.modulemap
0%: Write SwiftCLI.o.SwiftFileList (x86_64)
0%: Write SwiftCLI-OutputFileMap.json (x86_64)
0%: Write SwiftCLI.o.LinkFileList (x86_64)
0%: Write Rainbow-OutputFileMap.json (x86_64)
0%: Write Rainbow.o.SwiftFileList (x86_64)
0%: Write Rainbow.o.LinkFileList (x86_64)
0%: Write PathKit-OutputFileMap.json (x86_64)
0%: Write PathKit.o.LinkFileList (x86_64)
0%: Write PathKit.o.SwiftFileList (x86_64)
0%: Write Version.o.SwiftFileList (x86_64)
0%: Write Version.o.LinkFileList (x86_64)
0%: Copy Version.modulemap
0%: Copy SwiftCLI.modulemap
0%: Copy Rainbow.modulemap
0%: Copy PathKit.modulemap
0%: Compile Swift source files (x86_64)
0%: Compile Swift source files (x86_64)
0%: Compile Swift source files (x86_64)
0%: Compile Swift source files (x86_64)
0%: Compile PathKit.swift (x86_64)
0%: Compile Swift source files (x86_64)
0%: Compile Swift source files (x86_64)
0%: Compile Swift source files (x86_64)
26%: Copy Version-Swift.h (x86_64)
26%: Copy Version.swiftdoc (x86_64)
26%: Copy Version.swiftdoc (x86_64)
26%: Copy Version.swiftmodule (x86_64)
26%: Copy Version.swiftmodule (x86_64)
26%: Copy Version.swiftsourceinfo (x86_64)
26%: Copy Version.swiftsourceinfo (x86_64)
26%: Link Version.o (x86_64)
26%: Register execution policy exception for Version.o
34%: Copy Rainbow-Swift.h (x86_64)
34%: Copy Rainbow.swiftmodule (x86_64)
34%: Copy Rainbow.swiftdoc (x86_64)
34%: Copy Rainbow.swiftdoc (x86_64)
34%: Copy Rainbow.swiftmodule (x86_64)
34%: Copy Rainbow.swiftsourceinfo (x86_64)
34%: Copy Rainbow.swiftsourceinfo (x86_64)
34%: Link Rainbow.o (x86_64)
34%: Register execution policy exception for Rainbow.o
43%: Copy PathKit-Swift.h (x86_64)
43%: Copy PathKit.swiftdoc (x86_64)
43%: Copy PathKit.swiftmodule (x86_64)
43%: Copy PathKit.swiftdoc (x86_64)
43%: Copy PathKit.swiftmodule (x86_64)
43%: Copy PathKit.swiftsourceinfo (x86_64)
43%: Copy PathKit.swiftsourceinfo (x86_64)
43%: Link PathKit.o (x86_64)
43%: Register execution policy exception for PathKit.o
52%: Copy SwiftCLI.swiftdoc (x86_64)
52%: Copy SwiftCLI-Swift.h (x86_64)
52%: Copy SwiftCLI.swiftdoc (x86_64)
52%: Copy SwiftCLI.swiftmodule (x86_64)
52%: Copy SwiftCLI.swiftmodule (x86_64)
52%: Copy SwiftCLI.swiftsourceinfo (x86_64)
52%: Copy SwiftCLI.swiftsourceinfo (x86_64)
52%: Link SwiftCLI.o (x86_64)
52%: Write MintKit.modulemap
52%: Write MintKit-OutputFileMap.json (x86_64)
52%: Write MintKit.o.SwiftFileList (x86_64)
52%: Copy MintKit.modulemap
52%: Compile Swift source files (x86_64)
52%: Compile Swift source files (x86_64)
52%: Register execution policy exception for SwiftCLI.o
52%: Write MintKit.o.LinkFileList (x86_64)
66%: Copy MintKit.swiftmodule (x86_64)
66%: Copy MintKit-Swift.h (x86_64)
66%: Copy MintKit.swiftmodule (x86_64)
66%: Copy MintKit.swiftdoc (x86_64)
66%: Copy MintKit.swiftsourceinfo (x86_64)
66%: Copy MintKit.swiftsourceinfo (x86_64)
66%: Copy MintKit.swiftdoc (x86_64)
66%: Link MintKit.o (x86_64)
66%: Write MintCLI.modulemap
66%: Write MintCLI.o.SwiftFileList (x86_64)
66%: Write MintCLI-OutputFileMap.json (x86_64)
66%: Copy MintCLI.modulemap
66%: Compile Swift source files (x86_64)
66%: Compile Swift source files (x86_64)
66%: Register execution policy exception for MintKit.o
66%: Write MintCLI.o.LinkFileList (x86_64)
80%: Copy MintCLI.swiftmodule (x86_64)
80%: Copy MintCLI-Swift.h (x86_64)
80%: Copy MintCLI.swiftdoc (x86_64)
80%: Copy MintCLI.swiftmodule (x86_64)
80%: Copy MintCLI.swiftdoc (x86_64)
80%: Copy MintCLI.swiftsourceinfo (x86_64)
80%: Copy MintCLI.swiftsourceinfo (x86_64)
80%: Link MintCLI.o (x86_64)
80%: Write mint.SwiftFileList (x86_64)
80%: Write mint-OutputFileMap.json (x86_64)
80%: Compile Swift source files (x86_64)
80%: Compile main.swift (x86_64)
80%: Register execution policy exception for MintCLI.o
80%: Write mint.LinkFileList (x86_64)
91%: Link mint (x86_64)
91%: Copy Mint.swiftmodule (x86_64)
91%: Copy Mint.swiftdoc (x86_64)
91%: Copy Mint.swiftdoc (x86_64)
91%: Copy Mint.swiftsourceinfo (x86_64)
91%: Copy Mint.swiftsourceinfo (x86_64)
91%: Copy Mint.swiftmodule (x86_64)
98%: Generate mint.dSYM
98%: Register execution policy exception for mint
100%: Build succeeded
mkdir -p /usr/local/bin
cp -f .build/release/mint /usr/local/bin/mint
cp: .build/release/mint: No such file or directory
make: *** [install] Error 1
```

## environment

Xcode version: 12.1
macOS: 10.15.6
Git Commit: eaae1709570a0781b2572a2786fb9f57d2445f3c

--- a/Makefile
+++ b/Makefile
@@ -4,7 +4,7 @@ VERSION = 0.15.0
 
 PREFIX = /usr/local
 INSTALL_PATH = $(PREFIX)/bin/$(EXECUTABLE_NAME)
-BUILD_PATH = .build/release/$(EXECUTABLE_NAME)
+BUILD_PATH = .build/apple/Products/Release/$(EXECUTABLE_NAME)
 CURRENT_PATH = $(PWD)
 RELEASE_TAR = $(REPO)/archive/$(VERSION).tar.gz
 

